client/src/components/Footer.css

```diff
@@ -0,0 +1,65 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap');
```

Comment:
Again it would be good to have this at a higher level so you don't need to import them in multiple places